install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -272,6 +272,7 @@ spec:
       {{- end }}
       affinity:
       {{- include "gatewaynodeaffinity" (dict "root" $ "nodeSelector" $spec.nodeSelector) | indent 6 }}
+      {{- include "gatewaypodAntiAffinity" (dict "root" $ "podAntiAffinityLabelSelector" $spec.podAntiAffinityLabelSelector "podAffinityTermLabelSelector" $spec.podAffinityTermLabelSelector) | indent 6 }}
```

Comment:
This should be:
```
{{- include "gatewaypodAntiAffinity" (dict "podAntiAffinityLabelSelector" $spec.podAntiAffinityLabelSelector "podAffinityTermLabelSelector" $spec.podAffinityTermLabelSelector) | indent 6 }}
```

Since we don't need values from `$(root)` in our template file.

Reply:
I have removed "root" and seem to function okay using helm template